jormungandr-lib/src/time.rs

```diff
@@ -532,11 +529,9 @@ mod test {
 
     impl Arbitrary for LocalDateTime {
         fn arbitrary<G: Gen>(g: &mut G) -> Self {
-            // no nanoseconds for the local date time as this is not displayed
-            const NSECS: u32 = 0;
-
-            let secs = i64::arbitrary(g) % 0xFF_FFFF_FFFF;
-            LocalDateTime(Local.timestamp(secs, NSECS))
+            const MAX: i64 = ::time::Date::MAX.midnight().assume_utc().unix_timestamp();
```

Comment:
isn't this actually the beginning of the last day? (not that it matters though)

Reply:
yes, I was a bit lazy and didn't add the remaining hours. I won't be around to blame if something goes wrong in year 9999 anyway :)